eos/form-factors/observables.cc

```diff
@@ -2886,6 +2886,15 @@ namespace eos
                 make_form_factor_adapter("0->Kpi::Im{f_+}(Re{q2},Im{q2})", R"(\textrm{Im}(f_+^{0\to K\pi}(q^2)))",
                         &FormFactors<VacuumToPP>::im_f_p, std::make_tuple("Re{q2}", "Im{q2}")),
 
+                make_observable("0->Kpi::f_+(0)", R"(f_+^{0 \to K\pi}(0))", Unit::None(),
+                        &KSvD2025FormFactors<VacuumToKPi>::fp_at_0),
+
+                make_observable("0->Kpi::lambda_+", R"(\lambda_+^{('), K \to \pi})", Unit::None(),
```

Comment:
Inconsistent. The label contains a prime, and the next pseudoobservable is ``doubleprime``, but this has no ``prime`` in its name.

Reply:
Same goes for the name of the providing method.